### ISSUE TITLE
Binding ngModel to a number prevents component from rendering server side

### ISSUE THREAD

**Issue by eryi** [angular/universal#553]
- **I'm submitting a ...**
- [X] bug report
- **What modules are related to this Issue?**
- [X] express-engine
- **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem** by creating a github repo.
  1. git clone https://github.com/angular/universal-starter (49a9dc8)
  2. replace src/app/app.ts with the code inlined below (2 new lines added)
     1. `<input name="myName" [(ngModel)]="myVar">`
     2. `public myVar = 1234567;`
  3. npm install/npm start

```
import { Component } from '@angular/core';

@Component({
  selector: 'app',
  template: `
  <input name="myName" [(ngModel)]="myVar">
  <router-outlet></router-outlet>
  `
})
export class App {
  public myVar = 1234567;
}

@Component({
  selector: 'home',
  template: 'Home component'
})
export class Home {

}

```
- **What is the current behavior?**  
  1. The above component is rendered client side.
  2. By changing myVar from a number into a string (`public myVar = 123456;` into `public myVar = "123456";`) the bug no longer occurs and the component will be rendered server side.
- **What is the expected behavior?**
  1. Component should be rendered server side even when I bind ngModel to a number
- **What is the motivation / use case for changing the behavior?**
  1. The current behavior is very unexpected and hard to debug.
  2. Binding of ngModel to a number might be a common occurrence 
- **Please tell us about your environment:**
  - Angular version: 2.0.0
  - Browser: Chrome 51
  - Language: TypeScript 2.0
  - OS:  Linux x64
  - Platform:  NodeJs 6.2.1

**Comment by MarkPieszak** [comment]
@gdi2290 shouldn't this part we added a while back due to the trick here? 
https://github.com/angular/universal/blob/master/modules/platform-node/node-renderer.ts#L463

Not sure if ngModels are handled differently somehow...

**Comment by aarndt** [comment]
We were experiencing a similar issue with version ~2.0.10. We upgraded to 2.1.0-rc.1 and the issue seems to be resolved. In our case, it occurred when binding a numerical value to the [value] attribute on input and option elements.

@MarkPieszak The line that you linked to doesn't assign the value, as @gdi2290 indicated in a previous comment on the commit. I am curious as to how this was resolved. I see a [commit](https://github.com/angular/universal/commit/8b0a48d9162a9171cbec1ad9094c1c78bc8654fb) that only applies the value to the attribute if it is of type 'string', but I don't understand how that would have resolved our issue.
